pandas/tests/arrays/boolean/test_astype.py

```diff
@@ -20,7 +22,8 @@ def test_astype():
     tm.assert_numpy_array_equal(result, expected)
 
     result = arr.astype("str")
-    expected = np.array(["True", "False", "<NA>"], dtype="<U5")
+    endian = {"little": "<", "big": ">"}[byteorder]
```

Comment:
why don't we define this in https://github.com/pandas-dev/pandas/blob/main/pandas/_testing/__init__.py

e.g. make a module level ENDIAN and just use it here (and in other locations where you are doing similar things)

Reply:
Good plan. I’ll do that.